### PR TITLE
python@3.10 python-tk@3.10 3.10.11

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Review by carlocab — APPROVED** [review]

File: Formula/python@3.10.rb [hunk 1]
```diff
@@ -50,19 +49,24 @@ class PythonAT310 < Formula
               "bin/easy_install-3.7", "bin/easy_install-3.8", "bin/easy_install-3.9"
 
   # Always update to latest release
-  resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/5f/36/7374297692bb9dbd7569a0f84887c7e5e314c41d5d9518cb76fbb130620d/setuptools-67.2.0.tar.gz"
-    sha256 "b440ee5f7e607bb8c9de15259dba2583dd41a38879a7abc1d43a71c59524da48"
+  resource "flit-core" do
+    url "https://files.pythonhosted.org/packages/10/e5/be08751d07b30889af130cec20955c987a74380a10058e6e8856e4010afc/flit_core-3.8.0.tar.gz"
+    sha256 "b305b30c99526df5e63d6022dd2310a0a941a187bd3884f4c8ef0418df6c39f3"
```
Comment (carlocab) [inline Formula/python@3.10.rb:54]: This is bundled with Python now?

Comment (carlocab) [reply Formula/python@3.10.rb:54]: Ah, if I'm understanding https://github.com/pypa/wheel/pull/501 correctly: `flit-core` is only needed to build a wheel of `wheel`, correct?

If so, maybe we don't need to install it along with the formula?

Comment (branchvincent) [reply Formula/python@3.10.rb:54]: Yea that's right, `flit-core` is just a build time dependency and we only install it into the temporary `whl_build` venv (it doesn't end up in the formula)

Typically, `pip` would handle installing build dependencies but we've opted out with [`--no-build-isolation`](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-no-build-isolation)

Comment (carlocab) [reply Formula/python@3.10.rb:54]: Gotcha, we only do it during `install` and not `post_install`.

**Pull request merged**